src/Game/Graphic.cs

```diff
@@ -87,7 +87,7 @@ public int CompareTo(ushort other)
 
         public override string ToString()
         {
-            return $"0x{Value:X4}";
+            return $"{Value} (0x{Value:X4})";
```

Comment:
:bulb: I made this adjustment to both `Graphic` and `Hue` to aid in readability of these values, since they are often referred to either by their hex or decimal representation (for example, in UO:R, hues are generally referred to by their decimal representation).